### ISSUE TITLE
Access8math doesn't work in browsers

### ISSUE THREAD

**Issue by alisa-kalina** [tsengwoody/Access8Math#105]
Hello!
Unfortunately, I have the problem with addon in both chrome and Microsoft browsers. NVDA just speaks nothing when any MathML content appears. I've allready tried all NVDA versions from 2024.1 to 2024.3, all last versions of addon, but the effect is the same. I tested different Wikipedia articles and other websites. Befor I had a Laptop with Windows-8 and access8math worked without problems, but now on Windows-11 it doesn't read anything. I installed also Mathplayer and MathCat, changed access8math settings, but it didn't help. Below is the information I can see in Nvda.log file, and i didn't find any errors there.

INFO - globalCommands.script_navigatorObject_devInfo (18:49:11.964) - MainThread (7672):
Developer info for navigator object:
name: None
role: MATH
processID: 9100
roleText: None
states: 
isFocusable: False
hasFocus: False
Python object: <NVDAObjects.IAccessible.ia2Web.Math object at 0x06050050>
Python class mro: (<class 'NVDAObjects.IAccessible.ia2Web.Math'>, <class 'NVDAObjects.IAccessible.ia2Web.Ia2Web'>, <class 'NVDAObjects.IAccessible.IAccessible'>, <class 'NVDAObjects.window.Window'>, <class 'NVDAObjects.NVDAObject'>, <class 'documentBase.TextContainerObject'>, <class 'baseObject.ScriptableObject'>, <class 'baseObject.AutoPropertyObject'>, <class 'garbageHandler.TrackedObject'>, <class 'object'>)
description: None
location: RectLTWH(left=316, top=625, width=44, height=14)
value: ''
TextInfo: <class 'NVDAObjects.IAccessible.IA2TextTextInfo'>
appModule: AppModule(appModuleHandler, appName='chrome', processID=9100)
appModule.productName: 'Google Chrome'
appModule.productVersion: '131.0.6778.205'
appModule.helperLocalBindingHandle: c_long(126962072)
windowHandle: 1181376
windowClassName: 'Chrome_RenderWidgetHostHWND'
windowControlID: 33305824
windowStyle: 1445986304
extendedWindowStyle: 32
windowThreadID: 8744
windowText: 'Chrome Legacy Window'
displayText: ''
IAccessibleObject: <POINTER(IAccessible2) ptr=0x805ef74 at f4ab260>
IAccessibleChildID: 0
IAccessible event parameters: windowHandle=1181376, objectID=-4, childID=-2836
IAccessible accName: None
IAccessible accRole: ROLE_SYSTEM_EQUATION
IAccessible accState:  (0)
IAccessible accDescription: None
IAccessible accValue: ''
IAccessible2 windowHandle: 1181376
IAccessible2 uniqueID: -2836
IAccessible2 role: ROLE_SYSTEM_EQUATION
IAccessible2 states: IA2_STATE_OPAQUE (1024)
IAccessible2 attributes: 'display:math;tag:math;text-align:left;math:\n  <semantics>\n    <mrow class\\="MJX-TeXAtom-ORD">\n      <mstyle displaystyle\\="true" scriptlevel\\="0">\n        <mi>k</mi>\n        <mo>\\=</mo>\n        <mn>3</mn>\n      </mstyle>\n    </mrow>\n    <annotation encoding\\="application/x-tex">{\\\\displaystyle k\\=3}</annotation>\n  </semantics>\n;'
IAccessible2 relations:

**Comment by tsengwoody** [comment]
Are you using a Chromium-based browser? If so, the issue you're encountering is due to an update in Chromium, starting from version 131, which has impacted the support for MathML with assistive technologies on web platforms.
As a temporary solution, we recommend using Firefox as your browser for accessing math content on the web. Firefox continues to provide effective support for MathML and works seamlessly with Access8Math. For further details, you can check this issue and discussion:
https://github.com/nvaccess/nvda/issues/17421

**Comment by alisa-kalina** [comment]
> Are you using a Chromium-based browser? If so, the issue you're encountering is due to an update in Chromium, starting from version 131, which has impacted the support for MathML with assistive technologies on web platforms. As a temporary solution, we recommend using Firefox as your browser for accessing math content on the web. Firefox continues to provide effective support for MathML and works seamlessly with Access8Math. For further details, you can check this issue and discussion: [nvaccess/nvda#17421](https://github.com/nvaccess/nvda/issues/17421)

So, I installed the previous version of Chrome, 130. And now it works if i choose the MathCat in addon settings, but if I choose access8math it's still not speaking anything(( MathCat is great but it hasn't Russian language, that\'s why Access8math is better for me.

**Comment by tsengwoody** [comment]
Hello,
Have you already resolved the issue? By the way, please note that Chrome can now be updated to version 132.
Based on your description, when using Chrome version 130 the MathCat option in the add-on settings works fine, but the access8math option still isn't producing any speech. To help diagnose this issue further, we need to review your NVDA configuration.
Please provide the contents of the NVDA configuration file, nvda.ini, located in your NVDA user directory. On Windows, this file is typically located at:C:\Users\<YourUserName>\AppData\Roaming\nvda\nvda.ini
